documentation/CAMARA-Security-Interoperability.md

```diff
@@ -105,6 +106,45 @@ The following table defines the REQUIRED behaviour of the API Provider for the `
 | No                               | Supports DPoP                     | Bearer token         |
 | No                               | Requires DPoP                      | HTTP 400 `invalid_dpop_proof`<br>(see RFC [9449](https://www.rfc-editor.org/rfc/rfc9449.html#name-oauth-extensions-error-regi)) |
 
+
+#### Additional Recommendations for DPoP Implementations
+
+For API providers (e.g., Operators) that support and require the DPoP mechanism as defined in [RFC 9449](https://datatracker.ietf.org/doc/html/rfc9449), the following measures MAY be used to strengthen message integrity and replay protection. 
+
+**Request Binding to DPoP Proofs**
+
+As per RFC 9449 [section 11.7](https://www.rfc-editor.org/rfc/rfc9449.html#section-11.7), the DPoP proof binds the HTTP method (htm) and target URI (htu) but omits headers and the request body. To prevent message alteration or tampering, API providers SHOULD require clients to include additional CAMARA-defined claims in the DPoP proof: 
+
+|      Extension Claim   | Purpose  | Computation  | Encoding |
+|:-------------------:|:---------------------:|:-------------------:|:-------------------:|
+| x-camara:qh                          | Binds the query string to the DPoP proof | SHA-256 hash of the raw query string (if present) | Base64URL-encoded ([RFC 7515](https://www.rfc-editor.org/rfc/rfc7515.html) §2), without padding |
+| x-camara:bh                          | Binds the request body to the DPoP proof | SHA-256 hash of the full request body byte stream | Base64URL-encoded ([RFC 7515](https://www.rfc-editor.org/rfc/rfc7515.html) §2), without padding |
+
+The resulting hash values are included as string values within the DPoP proof JWT. Implementations MUST ignore any DPoP claims not defined in the base DPoP specification or in this CAMARA extension. 
+
+If a server requires DPoP Extensions, it MUST include the extended claims in the AS metadata to dynamically inform clients about these required claims. In addition, the server MAY advertise this extension claim requirements as part of the developer documentation with examples. Example Authorization Server Metadata (OIDC Discovery Document) typically as response to openIdConnectUrl: https://example.com/.well-known/openid-configuration
+
+json:
+
+{  
+  "issuer": "https://server.example.com", 
+   "token_endpoint": "https://server.example.com/oauth2/token” 
+   "authorization_endpoint": https://server.example.com/oauth2/authorize” 
+   "dpop_bound_access_tokens": true, 
+   "dpop_signing_alg_values_supported": ["ES256", “RS256”], 
+   "x-dpop_claims_required": [ "CAMARA:bh", "CAMARA:qh"]  
+// ... other OIDC fields  
+}  
+
+**Replay Protection Requirements**
+
+To improve replay protection beyond the baseline DPoP behavior: 
+
+- Each jti value MUST be unique per proof. 
```

Comment:
Maybe add a note that shows how the API provider could detect non-unique jtis? Is that line 145? Maybe make 143 and 145 one sentence?

What is the error message if the jti is not unique?